functions/go/fmt/metadata.yaml

```diff
@@ -0,0 +1,11 @@
+image: gcr.io/kpt-fn/fmt
+description: Format resources
+tags:
+  - mutator
+sourceURL: https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/master/functions/go/fmt
+examplePackageURLs:
+  - https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/master/examples/fmt/simple
+emails:
+  - mengqiy@google.com
```

Comment:
kpt-team@google.com for all 1st party functions

@mengqiy to keep this consistent

Reply:
https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/245